test/powershell/Modules/Microsoft.PowerShell.Management/Get-Content.Tests.ps1

```diff
@@ -99,13 +99,12 @@ baz
 "@
         $expected = 'foo'
         $tailCount = 3
-        [Microsoft.PowerShell.Commands.FileSystemCmdletProviderEncoding] $encoding = $EncodingName
 
         $testPath = Join-Path -Path $TestDrive -ChildPath 'TailWithEncoding.txt'
-        $content | Set-Content -Path $testPath -Encoding $encoding
+        $content | Set-Content -Path $testPath -Encoding $encodingName
```

Comment:
Should this test cover the ByteEncoding path?  It doesn't appear to.

Reply:
covered by the tests in engine/Basic